docs/non-interactive-install.md

```diff
@@ -171,3 +171,63 @@ The system hostname.
 If set to `true`, setpu will skip the final confirmation. If missing or set to
 any other value setup will display a summary and prompt for confirmation before
 anything is applied.
+
+## Where to put the answers.json file
+
+The `answers.json` file can be supplied in one of three ways.
+
+* On the install media
+* As a boot module via Loader
+* As a boot module via iPXE
+
+### On the install media
+
+The answer file may be placed at `private/answers.json` of the installation
+media. For USB media this is simply a matter of mounting the installer and
+copying the file. For ISO media, you will need to build your own ISO.
```

Comment:
For ISO media, could one put it in the proto/ area's directory, and then utter "gmake iso"? That doesn't feel right as I type this, but I think we should provide additional details here.

I'm happy to test this with before-and-after ISOs to confirm, but I'm not *exactly* sure where to put it in the proto area for an ISO build.  (I do have new-from-scratch VMs to test it with, however.)

Reply:
Also it's been a while since I've used USB keys... this drops in `$USBKEY_TOP_LEVEL/private/` ?

Reply:
> Also it's been a while since I've used USB keys... this drops in `$USBKEY_TOP_LEVEL/private/` ?

Yes.

Reply:
Per offline discussion, building ISOs with built-in answers.json is doable, but not easily so, and for now its likely wisest to point people at USB install media for those situations.